cli/templatepull_test.go

```diff
@@ -230,118 +230,93 @@ func TestTemplatePull_LatestStdout(t *testing.T) {
 
 // ToDir tests that 'templates pull' pulls down the active template
 // and writes it to the correct directory.
+// nolint: paralleltest
 func TestTemplatePull_ToDir(t *testing.T) {
-	t.Parallel()
-
-	client := coderdtest.New(t, &coderdtest.Options{
-		IncludeProvisionerDaemon: true,
-	})
-	owner := coderdtest.CreateFirstUser(t, client)
-	templateAdmin, _ := coderdtest.CreateAnotherUser(t, client, owner.OrganizationID, rbac.RoleTemplateAdmin())
-
-	// Create an initial template bundle.
-	source1 := genTemplateVersionSource()
-	// Create an updated template bundle. This will be used to ensure
-	// that templates are correctly returned in order from latest to oldest.
-	source2 := genTemplateVersionSource()
-
-	expected, err := echo.Tar(source2)
-	require.NoError(t, err)
-
-	version1 := coderdtest.CreateTemplateVersion(t, client, owner.OrganizationID, source1)
-	_ = coderdtest.AwaitTemplateVersionJobCompleted(t, client, version1.ID)
-
-	template := coderdtest.CreateTemplate(t, client, owner.OrganizationID, version1.ID)
-
-	// Update the template version so that we can assert that templates
-	// are being sorted correctly.
-	updatedVersion := coderdtest.UpdateTemplateVersion(t, client, owner.OrganizationID, source2, template.ID)
-	_ = coderdtest.AwaitTemplateVersionJobCompleted(t, client, updatedVersion.ID)
-	coderdtest.UpdateActiveTemplateVersion(t, client, template.ID, updatedVersion.ID)
-
-	dir := t.TempDir()
-
-	expectedDest := filepath.Join(dir, "expected")
-	actualDest := filepath.Join(dir, "actual")
-	ctx := context.Background()
-
-	err = extract.Tar(ctx, bytes.NewReader(expected), expectedDest, nil)
-	require.NoError(t, err)
-
-	inv, root := clitest.New(t, "templates", "pull", template.Name, actualDest)
-	clitest.SetupConfig(t, templateAdmin, root)
-
-	ptytest.New(t).Attach(inv)
+	// Prevents the tests from running in parallel.
```

Comment:
nit: you may want to add more context here, for instance ... _as the test changes the work dir._

Reply:
Question: Why can't this (parent) test run in parallel with other tests? I saw that the `os.Chdir` is removed, which is why I ask. I understand the subtests may conflict with each other, so at least there it's fine to not have parallel.

Also, whenever you add a nolint, it's good practice to include the why attached to it in the following format (unrelated example):

```
//nolint: paralleltest // Changes the environment.
```

Reply:
Good points, thanks.